packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -199,6 +200,11 @@ export function getDefaultPreferencesState() {
     useTokenDetection: true,
     smartTransactionsOptInStatus: false,
     useTransactionSimulations: true,
+    tokenSortConfig: {
```

Comment:
Should we add `tokenSortConfig` in type `PreferencesState` ?

Reply:
Added 👍